src/amber/cli/templates/app/config/routes.cr

```diff
@@ -7,6 +7,11 @@ Amber::Server.configure do |app|
     plug Amber::Pipe::Session.new
     plug Amber::Pipe::Flash.new
     plug Amber::Pipe::CSRF.new
+
+    # Reload clients browsers (development only)
+    if Amber.env.development?
```

Comment:
Does this logic work? Seems to be a conditional around a macro that generates code.

Reply:
I tried it and work as expected